doc/nrf/links.txt

```diff
@@ -1808,6 +1808,7 @@
 .. _`nRF54H20 SoC binaries v0.7.0 for EngB DKs`: https://files.nordicsemi.com/artifactory/SDSC/external/nrf54h20_soc_binaries_v0.7.0_engb.zip
 .. _`nRF54H20 SoC binaries v0.8.0`: https://files.nordicsemi.com/ui/native/SDSC/external/nrf54h20_soc_binaries_v0.8.0.zip
 .. _`nRF54H20 SoC binaries v0.9.2`: https://files.nordicsemi.com/ui/native/SDSC/external/nrf54h20_soc_binaries_v0.9.2.zip
+.. _`nRF54H20 SoC binaries v0.9.6`: https://artifactory.nordicsemi.no/artifactory/SDSC/external/nrf54h20_soc_binaries_v0.9.6.zip
```

Comment:
@ns-tolu The link does not seem to be working

Reply:
Is https://files.nordicsemi.com/ui/native/SDSC/external/nrf54h20_soc_binaries_v0.9.6.zip the correct one?